src/rendering/defaults.rs

```diff
@@ -36,8 +36,8 @@ pub fn get_sinc_vertices() -> Vec<Vertex> {
     let mut vertices = Vec::<Vertex>::new();
     for x_idx in -1000..1000 {
         for y_idx in -1000..1000 {
```

Comment:
Also, if you used constants for 1000 you would only have to change it once in the future if for some reason 1000 seems too small or too large.